src/backend/looped/self/ptfx.cpp

```diff
@@ -0,0 +1,39 @@
+#include "natives.hpp"
+#include "backend/looped_command.hpp"
+#include "gta/enums.hpp"
+#include "core/data/ptfx_effects.hpp"
+
+namespace big
+{
+
+	class ptfx_looped : looped_command
+	{
+		using looped_command::looped_command;
+
+		PedBones ptfx_bones[5] = {
+			PedBones::SKEL_Head,
+			PedBones::SKEL_L_Hand,
+			PedBones::SKEL_R_Hand,
+			PedBones::SKEL_L_Foot,
+			PedBones::SKEL_R_Foot
+		};
+
+        void show_ptfx_effect(const char* fx_name, const char* name)
+        {
+			for (const auto& ptfx_bone : ptfx_bones)
```

Comment:
mixed indentation (tabs and spaces)

Reply:
Forgot to fix that when I made my edits, I’ll let @Primexz handle that one, don’t wanna Pr off a Pr again

Reply:
I am currently working on the idea of @Seanghost117 .
I will fix the indents in the commit as well